scalatest/src/main/scala/org/scalatest/tools/SuiteDiscoveryHelper.scala

```diff
@@ -213,7 +214,7 @@ private[scalatest] object SuiteDiscoveryHelper {
     val wrapWithAnnotation = clazz.getAnnotation(classOf[WrapWith])
     if (wrapWithAnnotation != null) {
       val wrapperSuiteClazz = wrapWithAnnotation.value
-      val constructorList = wrapperSuiteClazz.getDeclaredConstructors()
+      val constructorList : Array[Constructor[_]] = wrapperSuiteClazz.getDeclaredConstructors().map(_.nn)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.